master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml

```diff
@@ -54,12 +54,19 @@ spec:
       labels:
         k8s-app: calico-etcd
       annotations:
+        # Mark this pod as a critical add-on; when enabled, the critical add-on scheduler
+        # reserves resources for critical add-on pods so that they can be rescheduled after
+        # a failure.  This annotation works in tandem with the toleration below.
         scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
@djosborne Just to close the loop, there used to be two annotations, this one to mark our pod as a critical add-on and a second one for the old format of toleration.  this one is still needed, but the toleration one has moved to the "real" toleration syntax.